srcpkgs/pulseaudio/template

```diff
@@ -76,3 +76,32 @@ pulseaudio-devel_package() {
 		vmove usr/share/vala
 	}
 }
+
+pulseaudio-utils_package() {
+	short_desc+=" - utilities"
+	pkg_install() {
+		vmove usr/bin/pa-info
+		vmove usr/bin/pacat
+		vmove usr/bin/pacmd
+		vmove usr/bin/pactl
+		vmove usr/bin/padsp
+		vmove usr/bin/pamon
+		vmove usr/bin/paplay
+		vmove usr/bin/parec
+		vmove usr/bin/parecord
+		vmove usr/bin/pasuspender
+		vmove usr/bin/pax11publish
+		vmove "usr/share/bash-completion/completions/*"
+		vmove usr/share/zsh/site-functions/_pulseaudio
+		vmove usr/share/man/man1/pacat.1
+		vmove usr/share/man/man1/pacmd.1
+		vmove usr/share/man/man1/pactl.1
+		vmove usr/share/man/man1/padsp.1
+		vmove usr/share/man/man1/pamon.1
+		vmove usr/share/man/man1/paplay.1
+		vmove usr/share/man/man1/parec.1
+		vmove usr/share/man/man1/parecord.1
+		vmove usr/share/man/man1/pasuspender.1
+		vmove usr/share/man/man1/pax11publish.1
```

Comment:
And use the loop above to install these: `vmove usr/bin/$b` and `vmove usr/share/man/man1/${b}.1`